lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -317,7 +318,13 @@ <h4 class="message-title">${_('You have already verified your ID!')}</h4>
             <h4 class="message-title">${_('Your verification will expire soon!')}</h4>
             ## Translators: start_link and end_link will be replaced with HTML tags;
             ## please do not translate these.
-            <p class="message-copy">${Text(_('Your current verification will expire before the verification deadline for this course. {start_link}Re-verify your identity now{end_link} using a webcam and a government-issued ID.')).format(start_link=HTML('<a href="{href}">'.format(href=reverse('verify_student_reverify'))), end_link=HTML('</a>'))}</p>
+            <p class="message-copy">${Text(_('Your current verification will expire before the verification deadline '
```

Comment:
Do we know that strings get scraped possibly in HTML files when the string is split across multiple lines? I know that they do not in JavaScript files.